view/statistics-flow-certificates.js

```diff
@@ -0,0 +1,322 @@
+'use strict';
+
+var _                 = require('mano').i18n.bind('View: Statistics')
+  , db                = require('../db')
+  , uncapitalize      = require('es5-ext/string/#/uncapitalize')
+  , location          = require('mano/lib/client/location')
+  , queryHandlerConf  = require('../apps/statistics/flow-query-conf')
+  , setupQueryHandler = require('../utils/setup-client-query-handler')
+  , getData           = require('mano/lib/client/xhr-driver').get
+  , memoize           = require('memoizee')
+  , copy              = require('es5-ext/object/copy')
+  , ObservableValue   = require('observable-value')
+  , assign            = require('es5-ext/object/assign')
+  , Pagination        = require('./components/pagination')
+  , modes             = require('../utils/statistics-flow-group-modes')
+  , selectService     = require('./components/filter-bar/select-service')
+  , selectCertificate = require('./components/filter-bar/select-certificate')
+  , selectPeriodMode  = require('./components/filter-bar/select-period-mode')
+  , itemsPerPage      = require('../conf/objects-list-items-per-page')
+  , isNaturalNumber   = require('es5-ext/number/is-natural')
+  , serviceQuery      = require('../apps-common/query-conf/service')
+  , certificateQuery  = require('../apps-common/query-conf/certificate');
+
+exports._parent        = require('./statistics-flow');
+exports._customFilters = Function.prototype;
+
+exports['flow-nav']                = { class: { 'submitted-menu-item-active': true } };
+exports['flow-by-certificate-nav'] = { class: { 'pills-nav-active': true } };
+
+var queryServer = memoize(function (query) {
```

Comment:
As we intend to reuse same url across pages, let's seclude it into outer util so it can be used by further views